hack/release-tools/tag-release.sh

```diff
@@ -99,8 +113,10 @@ if [[ "$VELERO_PATCH" > 0 ]]; then
 
```

Comment:
I can't specifically add a comment to line 112, but the "push" aspect of the warning at the end of the message now only applies in the case where `$publish` is true so it could be printed only in that case.